.buildkite/pipeline.ml.yml

```diff
@@ -572,4 +572,5 @@
     - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
     - (cd rllib_contrib/ddpg && pip install -r requirements.txt && pip install -e .)
     - ./ci/env/env_info.sh
-    - pytest rllib_contrib/ddpg/tests/test_ddpg.py
\ No newline at end of file
+    - pytest rllib_contrib/ddpg/tests/
```

Comment:
maybe check with Lonnie  to see if we need to create Bazel BUILD file for each algo?